index.html

```diff
@@ -8,6 +8,122 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="price">
+        <h2 class="visually-hidden">Наши тарифы</h2>
+        <table class="table">
+          <thead>
+            <tr class="table__row">
+              <th class="table__header"></th>
+              <th class="table__header">
+                <h2 class="table__plan-name">База</h2>
+                <p class="table__plan-price">1,99 USD</p>
+              </th>
+              <th class="table__header table__header_hit">
+                <h2 class="table__plan-name">Стандарт</h2>
+                <p class="table__plan-price">3,99 USD</p>
+              </th>
+              <th class="table__header">
+                <h2 class="table__plan-name">Анлим</h2>
+                <p class="table__plan-price">9,99 USD</p>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="table__row">
+              <td class="table__cell table__cell_content_name">
```

Comment:
Two things here:

1. The content of table cell can directly go inside of `td` tag. You don't have to use a separate `p` tag. If there are multiple paragraphs inside of a table cell, then using `p` tags inside of `td` tag, makes sense otherwise it's not needed.
2. The name of the key-value modifier - `table__cell_content_name` doesn't quite make sense. If we're using a normal BEM modifier, it should be sort of an adjective to either the element or block. When using Key-value Modifier, as in your case, the name should signify several options for an item's appearance, state, or behavior.

Reply:
@5hraddha I did it